config/src/lib.rs

```diff
@@ -19,6 +19,7 @@ use clap::{App, Arg, ArgMatches};
 
 // Include all "stolen" ripgrep code in this module
 mod ripgrep_config;
+pub mod tls_config;
```

Comment:
Wouldn't it make sense to have this in a separate package?

So stackable-common/common-config
And have this as a pure library which can be reused independently?